src/_scss/pages/agencyV2/statusOfFunds/_visualizationSection.scss

```diff
@@ -1,4 +1,10 @@
 .status-of-funds__visualization {
+    .status-of-funds__controls {
+        display: flex;
+        @media(min-width: $tablet-screen) {
```

Comment:
The mobile view starts as 992px so this should be $medium-screen instead of $tablet-screen